### DISCUSSION TITLE
Support for Ctrl-Ins, Shift-Ins and Shift-Del

### DISCUSSION THREAD

**Discussion by readonly24** [rustdesk/rustdesk#8741] [General]
Hello.

I would like to have support for copy, paste and cut with Ctrl-Ins, Shift-Ins and Shift-Del additionally to Ctrl-C, Ctrl-V and Ctrl-X

**Comment by fufesou** [comment]
I cannot understand clearly.

I think `Ctrl-Ins`, `Shift-Ins` and `Shift-Del`, these shortcuts should be supported by apps on the controlled side?

RustDesk just send the key events to the controlled side.
